### DISCUSSION TITLE
How to get order tracking information?

### DISCUSSION THREAD

**Discussion by kswanner** [abuzuhri/Amazon-SP-API-CSharp#253] [Q&A]
I need to be able to get the tracking information from an order, whether it was Amazon or us that shipped it. All I have to go with is the Amazon Order ID (3-7-7). I need the tracking number, carrier, method, and ship date. How can I get this information? I've looked into reports and other methods, but I can't find how to do this.

**Comment by abuzuhri** [comment]
Maybe its better to open this question here https://github.com/amzn/selling-partner-api-docs/discussions
